doc/developer_guide/transformations.rst

```diff
@@ -516,4 +516,72 @@ an if condition or a loop condition, and that intrinsic contains an array sectio
 then PSyclone may generate extra dependencies, which may hurt code performance. If
 this causes issues, please open an issue.
 
+Moving to the new transformations options
+=========================================
+PSyclone is currently moving from a dictionary of options to keyword arguments
+to control the `apply` and `validate` transformation methods. For example:
```

Comment:
Like in the other PR, I don't link how single backticks are rendered when referring to code, could you use double backticks everywhere in this section.

Reply:
Done